handlers/handler.go

```diff
@@ -0,0 +1,17 @@
+package handlers
+
+import (
+	runtime "github.com/Hammad887/chat-app"
+	"github.com/Hammad887/chat-app/docs/restapi/operations"
+)
+
+func NewCustomHandler(api *operations.ChatroomAPI, rt *runtime.Runtime) {
+	api.ServiceRegisterUserHandler = RegisterUserHandler(rt)
+	api.ServiceLoginHandler = LoginUserHandler(rt)
+	api.ServiceLogoutHandler = LogoutUserHandler(rt)
+	api.ServiceGetAllChatroomsHandler = GetAllChatroomsHandler(rt)
+	api.ServiceGetChatroomHandler = GetChatroomHandler(rt)
+	api.ServiceSendMessageHandler = SendMessageHandler(rt)
+	api.ServiceGetAllMessagesHandler = GetChatroomMessagesHandler(rt)
+	// add all new handlers
```

Comment:
you can use bearer token protocol to authenticate request

Reply:
My task was a bit different. It's requirement doesn't specify the authentication of token.

Reply:
are you saying your application does not support Token verification?